applications/launchpad/gui-react/src/components/CoinsList/index.tsx

```diff
@@ -6,12 +6,13 @@ import { CoinsListProps } from './types'
 
 const formatAmount = (amount: string) => {
   if (Number(amount) === 0) {
-    return '00 000'
+    return '00,000'
   } else {
-    // Add spaces to number
-    const splitted = amount.toString().split('.')
-    splitted[0] = splitted[0].replace(/\B(?=(\d{3})+(?!\d))/g, ' ')
-    return splitted.join('.')
+    try {
+      return Number(amount).toLocaleString()
```

Comment:
with values like "114 014.04480000018" being displayed I think we could add `maximumFractionDigits` and limit it to... 3 ? 5 ?

Reply:
Nice suggestion, thanks! I'm going to use `maximumFractionDigits: 2`. I suppose that we will work with integers (however not confirmed yet), and handling maximumFractionDigits is a good defense.